docs/ovs-pipeline.md

```diff
@@ -124,8 +124,8 @@ you will see these addresses show up in the OVS flows.
 ## Antrea Network Policy CRD Implementation
 
 In addition to the above tables created for K8s NetworkPolicy, Antrea creates
-additional dedicated tables to support the [ClusterNetworkPolicy](network-policy.md) CRD
-([CnpEgressRuleTables] and [CnpIngressRuleTables]).
+additional dedicated tables to support the [Antrea NetworkPolicy CRDs](network-policy.md)
+([AntreaPolicyEgressRuleTables] and [AntreaPolicyIngressRuleTables]).
 
 Consider the following ClusterNetworkPolicy in the Application tier as an
```

Comment:
maybe change it to: "Consider the following Antrea ClusterNetworkPolicy (ACNP) in the ..."

so that we introduce the ACNP acronym

Reply:
Done